autolink/postlink/activityLinker.js

```diff
@@ -1,23 +1,52 @@
 // @ts-check
-var path = require('./path');
+var path = require("./path");
 var fs = require("fs");
-var { warnn, logn, infon, debugn } = require("./log");
+var { errorn, warnn, logn, infon, debugn } = require("./log");
 
 class ActivityLinker {
   constructor() {
     this.activityPath = path.mainActivityJava;
+    this.extendNavigationActivity = false;
+    this.removeGetMainComponentName = false;
   }
 
   link() {
+    if (!this.activityPath) {
```

Comment:
This is one of the most basic checks, checking for the file that will be edited to exist in the given directory.